presto-main/src/main/java/com/facebook/presto/operator/scalar/IpPrefixFunctions.java

```diff
@@ -197,6 +197,54 @@ public static Block collapseIpPrefixes(@SqlType("array(IPPREFIX)") Block unsorte
         return blockBuilder.build();
     }
 
+    @Description("Split the input prefix into subnets the size of the new prefix length.")
+    @ScalarFunction("ip_prefix_subnets")
+    @SqlType("array(IPPREFIX)")
+    public static Block ipPrefixSubnets(@SqlType(StandardTypes.IPPREFIX) Slice prefix, @SqlType(StandardTypes.BIGINT) long newPrefixLength)
```

Comment:
static import StandardTypes.IPPREFIX and StandardTypes.BIGINT. 
I know the existing code didnt' use static import either, will you be able to add a second commit to make the change to all functions?

Reply:
Yes. I can send out a new PR that updates the rest of the functions.

Reply:
Actually, I think the static import of `StandardTypes.IPPREFIX` is in conflict with the static import of  `com.facebook.presto.type.IpPrefixType.IPPREFIX`.

Reply:
@yingsu00 Not a Java expert so not sure the best path forward given the naming conflict.